### ISSUE TITLE
Regarding Experimental Data

### ISSUE THREAD

**Issue by min-yang5** [Depth2World/NLOST#2]
Hello！

I would like to learn more about the detailed settings of the experimental data; for example, the distance between the object and the scattering surface, the time resolution, the scanning range, and other key data. Could the author kindly provide me with this information?

Thank you！

**Comment by Depth2World** [comment]
Sry for the late. Occupied with internship.
The distance varies in different scene. About 0.5m-2m. You can get the distance precisely through three views.
The time resolution is 32ps for FK and NLOST real-world data. The scanning range is 2m*2m ( For NLOST, 1.85m*1.85m pad to 2m*2m actually ).
Details about our self-built imaging system, please refer to read.me

**Comment by min-yang5** [comment]
Thank you for your answer.

I would like to ask if the three traditional recovery algorithms provided in the util-pytorch file support image recovery for non-confocal TOF data?

**Comment by Depth2World** [comment]
No, they are all for confocal data. For non-confocal data, you can follow <https://github.com/fmu2/nlos3d> for the pytorch version.
